2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,18 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
```

Comment:
This looks good to me.
One small point: try to keep an eye on indentation, as it will improve readability for other developers.